app/controllers/frontend_controller.rb

```diff
@@ -67,21 +63,21 @@ def session_is_secure?
   helper_method :sudo
 
   def sudo
-    return_path = request.xhr? ? request.fullpath : request.headers.fetch('Referer') { request.fullpath }
+    return_path = request.xhr? ? request.fullpath : request.headers.fetch("Referer") { request.fullpath }
```

Comment:
Single quotes and double quotes do not mean the same in Ruby. We should keep the single quotes in this case.

```suggestion
    return_path = request.xhr? ? request.fullpath : request.headers.fetch('Referer') { request.fullpath }
```

Reply:
Spending hours to config a good Ruby formatter in VSCode for this.. darn it! I'll fix all the quotes, not a voluntary change anyway.